BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/TraceExtractor/TraceSeverityLevelDimensionExtractor.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+    using System.Globalization;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.DataContracts;
+
+    internal class TraceSeverityLevelDimensionExtractor : IDimensionExtractor
+    {
+        public int MaxValues { get; set; } = 6;
+
+        public string DefaultValue { get; set; } = MetricTerms.Autocollection.Common.PropertyValues.Unspecified;
```

Comment:
I dont think this is not covered in unit test. lets add uT to test the scenario of Trace with no severity level specified.

Reply:
Done